unity-ggjj/Assets/Scripts/Actor/ActorController.cs

```diff
@@ -1,18 +1,15 @@
-using System;
 using System.Collections.Generic;
 using UnityEngine;
 using UnityEngine.Events;
-using UnityEngine.Serialization;
 
 public class ActorController : MonoBehaviour, IActorController
 {
+    [Tooltip("Drag a NarrativeScriptPlaylist here")]
```

Comment:
Nit: Mismatching tooltip text and property class.
```suggestion
    [Tooltip("Drag a DialogueController here")]
```